ServerCore/Helpers/TeamHelper.cs

```diff
@@ -136,5 +136,14 @@ public static async Task DeleteTeamAsync(PuzzleServerContext context, Team team)
 
             return new Tuple<bool, string>(true, "");
         }
+
+        /// <summary>
+        /// Returns whether or not a user is a full-time microsoft employee
+        /// </summary>
+        /// <returns>True if the user is a full-time microsoft employee and false otherwise</returns>
+        public static bool IsFTE(string email)
```

Comment:
Maybe IsNonIntern, since contractors are not FTEs...

Reply:
(though contractors are not eligible either so the logic is fine - the name and comment are slightly inaccurate though)